setup.py

```diff
@@ -16,7 +16,7 @@
         "isort==4.3.21",
         "flake8>=3.7.7,<4.0.0",
     ],
-    "dev": ["tox>=3.13.2,<4.0.0"],
+    "dev": ["tox>=3.13.2,<4.0.0", "Sphinx>=2.2.0,<3.0.0", "sphinx-rtd-theme"],
```

Comment:
It can be nice to move this to its own `doc` extra, like:

https://github.com/ethereum/trinity/blob/0ef04653c17d726463ae8a7f0ac01e0dcd930d54/setup.py#L85-L91

Reply:
Fixed in f7403ef349a03350224a21b19caf1343c54a35ec and added doc dependencies to the dev requirements in 817e9c88cfa61ef05d024ae8c7b2f4aaa7c4ca89